ivy/functional/frontends/torch/tensor.py

```diff
@@ -803,6 +803,13 @@ def matmul(self, other):
     def matrix_power(self, n, *, out=None):
         return torch_frontend.linalg.matrix_power(self, n, out=out)
 
+    def lstsq(self, other, rcond=None, driver=None):
+        return torch_frontend.linalg.lstsq(self, other, rcond, driver)
```

Comment:
I don't think this function exists in the torch frontend yet right? So we can't use `torch_frontend.linalg.lstsq`. We'll need to get that implemented before we can add these functions really